### PR TITLE
CB-9393: Documenting how to embed a SystemWebView

### PR DESCRIPTION
**Pull request by infil00p**
Redo of the pull request to update the SystemWebView documentation.

### CODE REVIEW

**Comment by infil00p** [comment]
@riknoll Bump, I've updated the branch

File: www/docs/en/dev/guide/platforms/android/webview.md [hunk 1]
```diff
@@ -29,106 +29,86 @@ components can communicate with each other, see Application Plugins.
...
+        private CordovaInterfaceImpl iface = new CordovaInterfaceImpl(this);
+
+1. Add the following to intialize the WebView
+
+        //Set up the webview
+        ConfigXmlParser parser = new ConfigXmlParser();
+        parser.parse(this);
+
+        SystemWebView webView = (SystemWebView) findViewById(R.id.tutorialView);
+        webInterface = new CordovaWebViewImpl(new SystemWebViewEngine(webView));
```
Comment (riknoll) [inline www/docs/en/dev/guide/platforms/android/webview.md]: `webInterface `needs to be declared

File: www/docs/en/dev/guide/platforms/android/webview.md [hunk 2]
```diff
@@ -29,106 +29,86 @@ components can communicate with each other, see Application Plugins.
...
-   `/framework/cordova-x.x.x.jar`.
+1. Create a project in Android Studio
 
-1. Copy the `.jar` file into the Android project's `/libs` directory.
+1. Copy the framework directory into your project's root directory and name it
+CordovaLib
+
+1. Add CordovaLib in your settings.gradle file
+
+include ':app', ':CordovaLib'
```
Comment (riknoll) [inline www/docs/en/dev/guide/platforms/android/webview.md]: This line needs to be indented

File: www/docs/en/dev/guide/platforms/android/webview.md [hunk 3]
```diff
@@ -29,106 +29,86 @@ components can communicate with each other, see Application Plugins.
...
+        webInterface = new CordovaWebViewImpl(new SystemWebViewEngine(webView));
+        webInterface.init(iface, parser.getPluginEntries(), parser.getPreferences());
+
+        webView.loadUrl(parser.getLaunchUrl());
+
+
+1. Copy the application's HTML and JavaScript files to the Android
+   project's `app/src/main/assets/www` directory.
+
+1. Copy the `config.xml` file from `/framework/res/xml` to the
```
Comment (riknoll) [inline www/docs/en/dev/guide/platforms/android/webview.md]: I found config.xml to be in `bin/templates/project/res/xml`

File: www/docs/en/dev/guide/platforms/android/webview.md [hunk 4]
```diff
@@ -29,106 +29,86 @@ components can communicate with each other, see Application Plugins.
...
+        parser.parse(this);
+
+        SystemWebView webView = (SystemWebView) findViewById(R.id.tutorialView);
+        webInterface = new CordovaWebViewImpl(new SystemWebViewEngine(webView));
+        webInterface.init(iface, parser.getPluginEntries(), parser.getPreferences());
+
+        webView.loadUrl(parser.getLaunchUrl());
+
+
+1. Copy the application's HTML and JavaScript files to the Android
```
Comment (riknoll) [inline www/docs/en/dev/guide/platforms/android/webview.md]: You also need to copy cordova.js from `bin/templates/project/assets/www` to that folder

File: www/docs/en/dev/guide/platforms/android/webview.md [hunk 5]
```diff
@@ -29,106 +29,86 @@ components can communicate with each other, see Application Plugins.
...
 SDK installed before you attempt the more unusual development option
-of embedding a WebView.  Starting with Cordova 1.9, the Android
-platform relies on a `CordovaWebView` component, which builds on a
-legacy `CordovaActivity` component that pre-dates the 1.9 release.
+of embedding a WebView.  Starting with Cordova 4.0, the Android
+platform relies on a `SystemWebView` component.
 
 1. To follow these instructions, make sure you have the latest Cordova
    distribution. Download it from
    [cordova.apache.org](http://cordova.apache.org) and unzip its
```
Comment (riknoll) [inline www/docs/en/dev/guide/platforms/android/webview.md]: The link is sort of hidden on cordova.apache.org. Might be easier to just give https://dist.apache.org/repos/dist/release/cordova/platforms/

**Comment by riknoll** [comment]
Minor comments, but otherwise looks pretty good!

**Comment by infil00p** [comment]
Blargh! I need to hunt down the merge conflict before re-sending this pull request. I have no idea what changed on this file, though.

File: www/docs/en/dev/guide/platforms/android/webview.md [hunk 6]
```diff
@@ -29,106 +29,86 @@ components can communicate with each other, see Application Plugins.
...
-                Config.init(this);
-                cwv.loadUrl(Config.getStartUrl());
-            }
+1. Add the following code to setup your Activity so that you have an interface
 
-1. If the application needs to use the camera, implement the
-   following:
+        private CordovaInterfaceImpl iface = new CordovaInterfaceImpl(this);
+
+1. Add the following to intialize the WebView
```
Comment (GustavoCostaW) [inline www/docs/en/dev/guide/platforms/android/webview.md]: initialize*

Comment (janpio) [reply www/docs/en/dev/guide/platforms/android/webview.md]: Commenting on old docs PRs is never the right solution. Better create a Stack Overflow question.

**Pull request closed without merging**